### PR TITLE
Make gulp work properly

### PR DESCRIPTION
**Pull request by stevemao**
typo
add default task
add missing callback arg
Also change `cb` to `callback` to make it consistent.
switch to libsass
Ruby sass is not working like this.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Needs a proper title and description.

**Comment by stevemao** [comment]
@sindresorhus not sure about the title. gulp does not work at all so basically I'm trying to fix it.

File: templates/common/root/_gulpfile.js [hunk 1]
```diff
@@ -76,8 +76,8 @@ gulp.task('lint:scripts', function () {
     .pipe(lintScripts());
 });
 
-gulp.task('clean:tmp', function (cb) {
-  rimraf('./.tmp', cb);
+gulp.task('clean:tmp', function (callback) {
+  rimraf('./.tmp', callback);
```
Comment (sindresorhus) [inline templates/common/root/_gulpfile.js]: No thanks. `cb` is the convention name in Node.

**Comment by stevemao** [comment]
Still problems and some depends on what tools and what configs you want to use.

**Pull request merged**